### ISSUE TITLE
Prepare for WOFpy Release v2.0.10-alpha

### ISSUE THREAD

**Issue by aufdenkampe** [ODM2/WOFpy#61]
Prepare for a new release in next few weeks.

The last release was Dec. 7, 2015 and since then there have been a number of critical commits to the master branch to better align WOFpy with the ODM2/ODM2PythonAPI, in particular @cdesyoun's recent commits cdesyoun@34038cced461f1b272ab3e96a48bd1357707b995 and cdesyoun@d1921a617b90203c5490ff768c33478e39876ebe which enabled WOFpy to work with [ODM2PythonAPI release v0.5.2-alpha](https://github.com/ODM2/ODM2PythonAPI/releases/tag/v0.5.2-alpha).

@valentinedwv, can you create a new release?
This would greatly assist @sreeder, @lsetiawan and @emiliom with testing their WOFpy deployments and eventually updating the Conda package at https://anaconda.org/odm2/repo

**Comment by emiliom** [comment]
As you prepare a new tagged release, please note the current mis-alignment of version labels/tags between the last github tagged release ("“v2.0.0009-alpha") and downstream pypi and conda packages ("2.0.9b0"). I'm copying notes about this that I just jotted down on our BiGCZ conference call notes from today:

> The conda package label is inconsistent with the github tagged release label! But according to the [conda recipes wofpy meta.yml file](https://github.com/ODM2/conda-recipes-ODM2/blob/master/recipes/wofpy/meta.yaml), the conda package was built using the [wofpy package on pypi](https://pypi.python.org/pypi/WOFpy/2.0.9b0), which is labelled 2.0.9b0 (like the conda package) and is dated 2015-12-07 (as the github tagged release). So, the conda package does seem to be based ultimately on the last tagged release from Dec 2015; the release version tag/label inconsistency was introduced in the creation of the pypi package from that tagged release.

**Comment by valentinedwv** [comment]
WOFPy and ODM2api uploaded

https://pypi.python.org/pypi/WOFpy

https://pypi.python.org/pypi/odm2api

**Comment by aufdenkampe** [comment]
@valentinedwv, what is your progress on finalizing this task, following the steps that we had discussed last week? These are:
* Preparing a GitHub Release of WOFpy, that is tagged "v2.0.10b" or perhaps "v2.0.10b1"
* Releasing this exact version on PyPi, pointing to this GitHub release (not the master branch), using the exact same version tag, and listing it as "beta" rather than "alpha" (to align with version tag)
* Emailing @emiliom & @ocefpaf so that they know to update the Conda recipes?

Also, we need you to work with @horsburgh & @sreeder to properly release https://github.com/ODM2/ODM2PythonAPI. 
* My understanding is that https://pypi.python.org/pypi/odm2api/0.5.2 was derived from the present head of the master branch, not from the latest release. We need the two to be exactly aligned. 
* It might make sense to re-release the master branch, with version "v0.5.3-alpha", to include your TravisCI commits from Oct. 4, then create an aligned release on PyPi with that exact version number and that points to that exact GitHub release.
* There is a, "odmtools" development branch of ODM2API that is substantially ahead of the master branch, so a new release may be in order. Please discuss with with @horsburgh & @sreeder.

**Comment by emiliom** [comment]
Regarding this:
> Emailing @emiliom & @ocefpaf so that they know to update the Conda recipes

@valentinedwv, just ping me via this github issue, when you let us know the github release is ready. I'll then follow up on conda packaging. No need for a separate email.

**Comment by valentinedwv** [comment]
revise numbering... code has 00010

**Comment by ocefpaf** [comment]
@emiliom and @lsetiawan are we OK to issue a new release? I have some time here at PDX before my flight and I can prepare the GitHub and conda releases.

PyPI only once I land b/c my password is in the other laptop :-(

**Comment by emiliom** [comment]
Nope, not yet. @lsetiawan is doing some additional work today and possibly tomorrow, and I was also hoping this release would include remaining enhancements to [wof/wofpy_config.py](https://github.com/ODM2/WOFpy/blob/master/wof/wofpy_config.py) by you, per [#72](72#issuecomment-296416841).

I'll follow up via email in a bit.

Also, this particular issue is probably outdated in terms of defining or tracking the new release ...

**Comment by ocefpaf** [comment]
> Also, this particular issue is probably outdated in terms of defining or tracking the new release ...

Yep. Let's close this one and follow the remaining of the work needed for the new release in a new issue.
